test/functional/page_objects/visualize_chart_page.ts

```diff
@@ -268,7 +268,7 @@ export function VisualizeChartPageProvider({ getService, getPageObjects }: FtrPr
         defaultFindTimeout * 2
       );
       return await Promise.all(
-        legendEntries.map(async chart => await chart.getAttribute('data-label'))
+        legendEntries.map(async (chart: any) => await chart.getAttribute('data-label'))
```

Comment:
<img width="1126" alt="visualize_chart_page" src="https://user-images.githubusercontent.com/10977896/74420827-59d78900-4e4c-11ea-9ac1-77afad1543ef.png">

I don't think we need to change the default `WebElementWrapper` type to `any`?

Reply:
I'll check